File: C++ Solutions/CodeChef/TrueandFalsePaper.cpp
#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int n;
	cin>>n;
	while(n>0){
	    int X,Y;
	    cin>>X>>Y;
	    int bobMark=X-Y;
	    cout<<bobMark<<"\n";
	    --n;
	}
	return 0;
}

File: JAVA Solutions/CodeChef/TrueandFalsePaper.java
/* package codechef; // don't place package name! */

import java.util.*;
import java.lang.*;
import java.io.*;

/* Name of the class has to be "Main" only if the class is public. */
class TrueandFalsePaper
{
	public static void main (String[] args) throws java.lang.Exception
	{
		// your code goes here
		Scanner read=new Scanner(System.in);
		int n=read.nextInt();
		while(n>0){
		    int X=read.nextInt();
		    int Y=read.nextInt();
		    int bobMarks=X-Y;
		    System.out.println(bobMarks);
		    --n;
		}
		read.close();
	}
}
